Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -333,6 +325,35 @@ bool Digitizer::convertHits(const int det, const std::vector<HitType>& hits, TRD
       // The distance of the position to the middle of the timebin
       double timeOffset = ((float)timeBinTruncated + 0.5 - timeBinIdeal) / samplingRate;
 
+      // ************************************************************
+      // Check if there are signals already
+      //  - Should we move this block? It's only used here
+      //  - It's only used here...
+      // ************************************************************
+      std::vector<int> pads_in_response(kNpad);
+      std::vector<int> keys_in_response(kNpad);
```

Comment:
consider not doing local std::vector allocation inside a loop. You could create these containers outside and reuse them.

Reply:
There two vectors are removed after this first code review iteration.